SmartPlaces.Facilities/lib/OntologyMapper/test/OntologyMapper.Test.csproj

```diff
@@ -18,15 +17,19 @@
 
 	<ItemGroup>
 		<PackageReference Include="coverlet.collector" Version="3.1.0" />
-		<PackageReference Include="Microsoft.Azure.DigitalTwins.Parser" Version="6.3.3-preview" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.1" />
+		<PackageReference Include="Divergic.Logging.Xunit" Version="4.0.0" />
+		<PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
+		<PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.8" />
+		<PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-		<PackageReference Include="Moq" Version="4.18.2" />
+		<PackageReference Include="Moq" Version="4.17.2" />
```

Comment:
Just curious why this is rolling backwards?

Reply:
Unintentional. Fixed